esmvaltool/recipes/ref/recipe_portrait_regions.yml

```diff
@@ -0,0 +1,322 @@
+# ESMValTool
+---
+documentation:
+  title: Performance metrics plots for several regions using extract_shape.
+  description: >
+    Compare performance of simulations to a reference dataset.
+  authors:
+    - cammarano_diego
+    - lindenlaub_lukas
+    - weigel_katja
+  maintainer:
+    - lindenlaub_lukas
+  references:
+    - eyring21ipcc
+    - gleckler08jgr
+
+datasets:
+  # cmip6
+  # - {dataset: ACCESS-ESM1-5, institute: CSIRO}
+  - {dataset: MPI-ESM1-2-HR}
+
+preprocessors:
+  default: &default  # common preprocessor settings
+    regrid:
+      target_grid: 1x1
+      scheme: linear
+    distance_metric:
+      metric: weighted_rmse
+    climate_statistics:
+      operator: mean
+      period: month
+    mask_fillvalues:
+      threshold_fraction: 0.95
+  ECA:
+    <<: *default
+    extract_shape:
+      shapefile: ar6
+      method: contains
+      ids: {Acronym: [ECA]}
+  ENA:
+    <<: *default
+    extract_shape:
+      shapefile: ar6
+      method: contains
+      ids: {Acronym: [ENA]}
+  MED:
+    <<: *default
+    extract_shape:
+      shapefile: ar6
+      method: contains
+      ids: {Acronym: [MED]}
+  SAH:
+    <<: *default
+    extract_shape:
+      shapefile: ar6
+      method: contains
+      ids: {Acronym: [SAH]}
+  WCE:
+    <<: *default
+    extract_shape:
+      shapefile: ar6
+      method: contains
+      ids: {Acronym: [WCE]}
+
```

Comment:
I think we should have all the regions here, not just the five that are listed here. Since this is a super interesting diagnostic for the CORDEX community.

Reply:
It is no problem to add more regions, but it will make the plot an the recipe huge. But I think it is important to first decide, what kind of portrait plot we want (see my answer to @bouweandela below)?